uktrade/data-hub-frontend
# Reactify link global HQ page

## Description of change
The page for linking a global HQ has been reactified using the same pattern as #5328.

## Test instructions

With your FE pointing to dev, go to [this page](http://localhost:3000/companies/bbab3b56-d2bb-4d5e-bb2f-dd159e5439dc/hierarchies/ghq/search). It should be possible to find a Global HQ and link it.

## Screenshots

![Screenshot 2023-06-23 at 12-21-30 Department for Business and Trade](https://github.com/uktrade/data-hub-frontend/assets/36161814/9d34d97f-a5d7-4f10-9159-66f8cf6211a7)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
